### PR TITLE
Add default_auto_field in apps config

### PR DESCRIPTION
**Pull request by peymanslh**
~~Add DEFAULT_AUTO_FIELD in the examples setting.~~
Add default_auto_field in apps config.

Closes: #436

### CODE REVIEW

**Comment by claudep** [comment]
I'd suggest https://github.com/jazzband/django-two-factor-auth/issues/436#issuecomment-1445363150 instead.

**Review by claudep — CHANGES_REQUESTED** [review]
Thanks! However I think we should only add it to apps that have models.

**Comment by claudep** [comment]
You may also include a note in the Changelog (in a new Unreleased section).

File: two_factor/apps.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 class TwoFactorConfig(AppConfig):
     name = 'two_factor'
     verbose_name = "Django Two Factor Authentication"
+    default_auto_field = "django.db.models.AutoField"
```
Comment (claudep) [inline two_factor/apps.py:8]: Is this needed? I think the `two_factor` app has no longer models.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+### Fixed
+- Add `default_auto_field` in apps config.
```
Comment (claudep) [inline CHANGELOG.md:5]: I would try to explain the change in more details (defaulting to AutoField...)

File: two_factor/apps.py (hunk 1 above)
Comment (peymanslh) [reply two_factor/apps.py:8]: I saw that there are no models in the `two_factor` app but there was a migration directory in it and it was confusing for me.

File: CHANGELOG.md (hunk 2 above)
Comment (peymanslh) [reply CHANGELOG.md]: Fixed it.

File: two_factor/apps.py (hunk 1 above)
Comment (peymanslh) [reply two_factor/apps.py]: Fixed it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+### Fixed
+- Set `default_auto_field` to `AutoField` in apps config that have models for Django>=3.2.
```
Comment (claudep) [inline CHANGELOG.md:5]: I would write: `Set default_auto_field to AutoField in apps config that have models, so no migrations are generated for projects defaulting to BigAutoField`. What do you think?

Comment (claudep) [reply CHANGELOG.md:5]: Oh and I would also mention the ticket number at the end of the note `(#436)`

**Comment by peymanslh** [comment]
@claudep 
Thanks for your review and suggestion

**Review by claudep — APPROVED** [review]
Thanks!

**Pull request merged**